package.json

```diff
@@ -43,6 +43,7 @@
     "react-dev-utils": "^5.0.1",
     "react-dom": "^16.4.0",
     "react-notify-toast": "^0.5.0",
+    "react-onclickoutside": "^6.7.1",
```

Comment:
This package, `react-onclickoutside`, does not seem to be used anywhere in the codebase, is there something I am missing?

Reply:
@IAMOTZ, thanks for pointing this out, I have removed this package since I was not using it anywhere.